token_auth/serializers/login.py

```diff
@@ -0,0 +1,46 @@
+from rest_framework import serializers
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import User
+from rest_framework.exceptions import AuthenticationFailed
+from django.contrib.auth.tokens import PasswordResetTokenGenerator
+from rest_framework_simplejwt.tokens import RefreshToken, TokenError
+from django.contrib import auth
+from django.utils.encoding import smart_str, force_str, smart_bytes, DjangoUnicodeDecodeError
+from django.utils.http import urlsafe_base64_decode, urlsafe_base64_encode
+
+
+class LoginSerializer(serializers.ModelSerializer):
```

Comment:
Can you please add doc strings here, giving a brief explanation of what this class is for?

Reply:
Done